README.rst

```diff
@@ -226,6 +235,10 @@ feature kræver det følgende afhængigheder::
 
   $ sudo apt install libxmlsec1-dev libxmlsec1-openssl postgresql-contrib
 
+For at få datamodellen i LoRas testinstans til at afspejle datamodellen i MO 
+vil MO's: ``os2mo/setup/db_structure.py`` overskrive oio_rests: ``oio_common/db_structure.py``
+i det aktuelle python-environment.
```

Comment:
```suggestion
i det aktuelle pythonmiljø.
```

I øvrigt er det forhåbentligt ikke tilfældet længere — jeg har tilrettet `test_support.py` så det ikke burde være nødvendigt. Så længe denne ændring overskriver kildekode i pythonmiljøet får den altså et :-1: herfra.

Reply:
Jeg fik ikke dine ændringer desangående til at fungere for mig - husk - det eker på jenkins

Reply:
I respektfully disagree on using 'pythonmiljø' - this word only gets 3 hits on goole. 
Python-environment should be the preferred term in danish too.

Reply:
Det hedder "pythonmiljø", hvis det hedder "udviklingsmiljø" - og "python-environment", hvis det hedder "udviklings-environment".

Det er altså det første, der er rigtigt.